modules/base/rendering/renderablemodel.cpp

```diff
@@ -390,9 +463,22 @@ RenderableModel::RenderableModel(const ghoul::Dictionary& dictionary)
         _modelTransform = glm::mat4_cast(glm::quat(glm::radians(_rotationVec.value())));
```

Comment:
Same here re `.value()`

Reply:
I tried to take away the `.value()` here but it didn't work :(